ivy/functional/frontends/paddle/nn/functional/loss.py

```diff
@@ -230,3 +230,56 @@ def margin_ranking_loss(input, other, label, margin=0.0, reduction="mean", name=
     out = ivy.atleast_1d(out)
 
     return out
+
+
+@with_supported_dtypes({"2.5.1 and below": ("float32", "float64")}, "paddle")
+@to_ivy_arrays_and_back
+def triplet_margin_loss(
+    input,
+    positive,
+    negative,
+    margin=1.0,
+    p=2.0,
+    eps=1e-06,
+    swap=False,
+    reduction="mean",
+):
+    def pairwise_distance(x1, x2, *, p=2.0, eps=1e-06, keepdim=False):
+        x1, x2 = paddle.promote_types_of_paddle_inputs(x1, x2)
+        x1_dim = len(x1.shape)
+        x2_dim = len(x2.shape)
+        if x1_dim > x2_dim:
+            output_dim = x1_dim
+        else:
+            output_dim = x2_dim
+
+        return ivy.vector_norm(
+            x1 - x2 + eps, ord=p, axis=output_dim - 1, keepdims=keepdim
+        )
+
+    reduction = _get_reduction_func(reduction)
+
+    a_dim = input.ndim
+    p_dim = positive.ndim
+    n_dim = negative.ndim
+
+    ivy.assertions.check_true(
+        a_dim == p_dim and p_dim == n_dim,
+        lambda: (
+            "The anchor, positive, and negative tensors are expected to have "
```

Comment:
This should be renamed to `input`
```suggestion
            "The input, positive, and negative tensors are expected to have "
```